openfisca_us/variables/gov/states/mo/tax/income/mo_income_tax_before_credits.py

```diff
@@ -0,0 +1,16 @@
+from openfisca_us.model_api import *
+
+
+class mo_income_tax_before_credits(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "MO income tax before credits"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://dor.mo.gov/forms/MO-1040%20Print%20Only_2021.pdf"
+
+    def formula(tax_unit, period, parameters):
+        agi = tax_unit("adjusted_gross_income", period)
+        rates = parameters(period).gov.states.mo.tax.income.rates
+        tax_due = rates.calc(agi, right=True)
```

Comment:
Marginal rate scales aren't affected by the `right` arg.
```suggestion
        return rates.calc(agi)
```

Reply:
got it, makes sense, thanks!